ui.frontend/src/main/webpack/site/js/components/searchInput.ts

```diff
@@ -18,10 +18,9 @@ type SearchInputOptions = {
   autoSuggestionDebounceTime: number
   searchContainer: HTMLDivElement
 }
-const SUGGESTION_DROPDOWN_ID = 'suggestions'
+const SUGGESTION_DROPDOWN_ID = 'cmp-saas-suggestions'
 const SEARCH_INPUT_CLASS = 'cmp-saas__search-input'
-const SAAS_CONTAINER_FORM_SUGGESTIONS_CLASS =
-  '.cmp-saas__form .cmp-saas__suggestions'
+const SAAS_CONTAINER_FORM_SUGGESTIONS_CLASS = `.cmp-saas__form #${SUGGESTION_DROPDOWN_ID}`
```

Comment:
i would update the const name to ...._SELECTOR or if you have smth else in mind.